lib/dispatcher.mjs

```diff
@@ -5,6 +5,7 @@ export const createDispatcher = (clientId, producer) => ({
     { partition = null, key = null, ts = Date.now() } = {}
   ) => {
     console.log(`${clientId} / Producer.Dispatcher.Dispatch('${topic}')`);
+    console.log("message", message.toString());
```

Comment:
This might lead to a very verbose log, we should put it under a `debug` condition or something.

```
if (LOG_LEVEL === 'debug') {
  // do verbose logging here
}
```